packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -105,6 +121,7 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
 
   const screenShareButtonStyles = useMemo(
     () => mergeButtonBaseStyles(checkedButtonOverrideStyles(theme, screenShareButtonProps.checked)),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
we should avoid disabling the linting as much as possible - if someone changes code in the future it hides potential render errors (I've fixed a bug recently that was caused by this).

If `checkedButtonOverrideStyles` only needs themePrimary it should probably just take in that prop